pycsob/client.py

```diff
@@ -17,6 +20,202 @@ def send(self, request, **kwargs):
         return super(HTTPAdapter, self).send(request, **kwargs)
 
 
+class ConvertMixin:
+    """Convert instance into ordered dict."""
+    
+    def _to_camel_case(self, name: str) -> str:
+        """Convert name to camel case form."""
+        parts = name.split("_")
+        return parts[0] + "".join(i.title() for i in parts[1:])
+
+    def _fields(self) -> Tuple[str, ...]:
+        return (i.name for i in fields(self))
+
+    def to_dict(self) -> Dict[str, Union[str, Dict]]:
+        data = []
+        for name in self._fields():
+            value = getattr(self, name)
+            if value in conf.EMPTY_VALUES:
+                continue
+            if hasattr(value, 'to_dict'):
+                value = value.to_dict()
+            data.append((self._to_camel_case(name), getattr(self, f"_format_{name}", lambda v: v)(value)))
+        return OrderedDict(data)
+
+
+@dataclass
+class CartItem(ConvertMixin):
+    """Cart item for creating card payment."""
+    # Documentation: https://github.com/csob/paymentgateway/wiki/Basic-methods#item---cart-item-object-cart-
+
+    name: str
+    quantity: int
+    amount: int
+    description: Optional[str] = None
+    
+    def _format_name(self, value: str) -> str:
+        return value[:20]
+    
+    def _format_description(self, value: str) -> str:
+        return value[:40]
+
+
+@dataclass
+class CustomerAccount(ConvertMixin):
+    """Customer account data."""
+    # Documentation: https://github.com/csob/paymentgateway/wiki/Purchase-metadata#customeraccount-data-
+
+    created_at: Optional[str] = None
```

Comment:
Wouldn't it be more appropriate to use the same names in the same order as stated in the documentation? Why do we have to transpile?

Reply:
Please see above for my reasons.

Reply:
Same to me. We'll simplify the code considerably. That's where a little inconsistency pays off, right?